packages/snaps-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.3.0]
```

Comment:
Not sure what to choose between a minor or a patch version :thinking:

Reply:
I think this should be a patch, and the section below changed to "Fixed".